Resources/Prototypes/Maps/arena.yml

```
@@ -1,76 +1,79 @@
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Arena Station {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'DV'
+        - type: StationEmergencyShuttle
+          emergencyShuttlePath: /Maps/Shuttles/DeltaV/NTES_UCLB.yml
+        - type: StationJobs
+          overflowJobs:
+            - Passenger
+          availableJobs:
+          #civilian
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)